sdk/digitaltwins/Azure.DigitalTwins.Core/tests/ComponentTests.cs

```diff
@@ -39,7 +39,7 @@ public async Task Component_Lifecycle()
                 // create wifi model
                 string roomWithWifiModel = TestAssetsHelper.GetRoomWithWifiModelPayload(roomWithWifiModelId, wifiModelId, wifiComponentName);
 
-                await client.CreateModelsAsync(new List<string> { roomWithWifiModel, wifiModel }).ConfigureAwait(false);
+                await createAndListModelsAsync(client, new List<string> { roomWithWifiModel, wifiModel }).ConfigureAwait(false);
```

Comment:
This is javaCasing bro :p

Reply:
Tim and Azad are getting into me with this whole Java thing..sigh